common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -2147,7 +2147,7 @@ object CelebornConf extends Logging {
       .doc("Timeout for a Celeborn worker to commit files of a shuffle. " +
         "It's recommended to set at least `240s` when `HDFS` is enabled in `celeborn.storage.activeTypes`.")
       .version("0.3.0")
-      .fallbackConf(RPC_ASK_TIMEOUT)
+      .fallbackConf(NETWORK_TIMEOUT)
```

Comment:
I assume the default value should short than NETWORK_TIMEOUT, otherwise the request may fail by network timeout

Reply:
how about explicitly setting it 180s?

Reply:
I think you are right. I just increased the timeout and max retries.